packages/venia-ui/lib/components/Password/password.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
Not using this component in the create account component because it will be handled in its own ticket.